jfrog-cli/utils/tests/utils.go

```diff
@@ -279,13 +279,18 @@ func (m *gitManager) execGit(args ...string) (string, string, error) {
 	return strings.TrimSpace(stdout.String()), strings.TrimSpace(stderr.String()), err
 }
 
-// Need to prepare the environment, for example such as .git directory and the config, head files.
-// Renaming the already prepared directories.
-func PrepareDotGitDir(t *testing.T, path string, setToParentDir bool) (string, string) {
-	baseDir := GetBaseDir(setToParentDir)
+// Prepare the .git environment for the test. Takes an existing folder and making it .git dir.
+// path - the dir to change to .git
+// targetDir - folder containing the path, usually "testdata".
+// switchToParentDir - true if need to look for 'targetDir' in the parent of current working dir.
```

Comment:
Suggestion - instead of passing the switchToParentDir boolean, we can perhaps send an actual relative path. Instead of passing *true*, we can pass ".." (the relative path to the parent directory).
The *switchToParentDir* is not very intuitive.
Same goes for the *GetBaseDir* function.

Reply:
Removed the _GetBaseDir_ function, changed _PrepareDotGitDir_ function so it will be easier to understand how to use it.